manifests/profiles/ambient.yaml

```diff
@@ -6,6 +6,7 @@ spec:
     defaultConfig:
       proxyMetadata:
         ISTIO_META_ENABLE_HBONE: "true"
+        ISTIO_META_LISTEN_QBONE: "true"
```

Comment:
afaik QBONE is already a named thing and is *not* what we are doing - it is directly using QUIC without HTTP/3

Reply:
It's no worse than HBONE... For our purposes, it just means QUIC transport which can be either HTTP3 CONNECT or MASQUE whenever that's ready.